### PR TITLE
Flash manually downloaded OEM binaries on Suzu

### LINKED ISSUE #37: Update Suzu config to use manual_download step and OEM fastboot flash
With the addition of https://github.com/ubports/ubports-installer/issues/1377, the suzu.json config needs to be updated to make use of the manual_download step for the OEM binary and the follow-on fastboot flash oem to install it.

The specific OEM for Suzu is: SW_binaries_for_Xperia_AOSP_N_MR1_5.7_r1_v08_loire.zip (named "Software binaries for AOSP Nougat (Android 7.1) – Kernel 4.4 – Loire")
SHA256: 9d9d6df01f0816e74b386daf82658768a54e515161ee9fb907a7aae292167e51
The download location is: https://developer.sony.com/file/download/software-binaries-for-aosp-nougat-android-7-1-kernel-4-4-loire/

### PR DESCRIPTION
**Pull request by retro486**
Updates suzu.json config to make use of new manual_download functionality and flash in a working order. Fixes #37

### CODE REVIEW

File: v1/suzu.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "codename": "suzu",
   "unlock": ["confirm_model", "upgrade_android"],
   "user_actions": {
+
```
Comment (NeoTheThird) [inline v1/suzu.json:6]:
```suggestion
```

File: v1/suzu.json [hunk 2]
```diff
@@ -77,6 +91,23 @@
...
+              "group": "firmware",
+              "raw": true
+            }
+          ]
+        },
+        {
+          "type": "fastboot:erase",
+          "condition": {"var": "bootstrap", "value": true},
+          "partition": "system"
+        },
```
Comment (NeoTheThird) [inline v1/suzu.json:110]: Is this needed?

File: v1/suzu.json [hunk 3]
```diff
@@ -77,6 +91,23 @@
...
         },
+        {
+          "type": "fastboot:flash",
+          "condition": {"var": "bootstrap", "value": true},
+          "flash": [
+            {
+              "partition": "oem",
+              "file": "SW_binaries_for_Xperia_AOSP_N_MR1_5.7_r1_v08_loire.img",
+              "group": "firmware",
+              "raw": true
```
Comment (NeoTheThird) [inline v1/suzu.json:102]: didn't you say you did not need raw?

File: v1/suzu.json (hunk 2 above)
Comment (NeoTheThird) [reply v1/suzu.json:110]: If not, you might be able to merge the flashing steps

**Review by NeoTheThird — CHANGES_REQUESTED** [review]
Thanks! Just some questions.

File: v1/suzu.json (hunk 2 above)
Comment (retro486) [reply v1/suzu.json:110]: It turns out it is required to wipe the system partition; this was one of the two missing pieces to get this sucker working.

File: v1/suzu.json (hunk 3 above)
Comment (retro486) [reply v1/suzu.json]: D'oh. Wrong copy open in my editor. Updating now. Do I cancel this PR and create a new one?

File: v1/suzu.json (hunk 1 above)
Comment (retro486) [reply v1/suzu.json]: Done!

File: v1/suzu.json (hunk 3 above)
Comment (retro486) [reply v1/suzu.json]: Applied correct changes to my branch in this PR.

**Review by NeoTheThird — COMMENTED** [review]
Awesome, good job! Thank you. Can be merged as soon as [0.5.6-beta](https://github.com/ubports/ubports-installer/releases/0.5.6-beta) is released.

**Comment by NeoTheThird** [comment]
cf also https://github.com/ubports/ubports-installer/issues/1377 and https://github.com/ubports/ubports-installer/issues/1305

**Comment by retro486** [comment]
I think the system erase step will fix the boot loop at recovery issue others were having (it did for me) and the oem flash without the raw flag will fix the stuck at Sony logo issues... I don't have any other Xperias to test though.

**Comment by NeoTheThird** [comment]
@retro486 yep, sorry, my comment was just for future reference so there's a note in the related issues that this is in the works :) i should have clarified

**Pull request merged**